pom.xml

```diff
@@ -18,6 +18,10 @@
             <id>public.onebusaway.org</id>
             <url>http://nexus.onebusaway.org/content/groups/public/</url>
         </repository>
+        <repository>
```

Comment:
Interesting, I didn't know that SQLite isn't officially supported as a hiberate dialect.  Looks like MySQL and PostGIS are supported:
https://developer.jboss.org/wiki/SupportedDatabases2?_sscc=t

In theory I believe we should be able to easily switch database providers - do you know of any downsides to switching to an officially supported dialect for a provider like MySQL?